README.md

```diff
@@ -18,26 +17,66 @@ in Python without C++.
 
 Halide requires C++17 (or later) to use.
 
-For more detail about what Halide is, see http://halide-lang.org.
+For more detail about what Halide is, see https://halide-lang.org.
 
-For API documentation see http://halide-lang.org/docs
+For API documentation see https://halide-lang.org/docs.
 
-To see some example code, look in the tutorials directory.
+For some example code, read through the tutorials online
+at https://halide-lang.org/tutorials. The corresponding code is in the
+`tutorials/` directory. Larger examples are in the `apps/` directory.
 
 If you've acquired a full source distribution and want to build Halide, see the
-[notes below](#building-halide-with-cmake).
+[notes below](#building-halide).
 
 # Getting Halide
 
+## Pip
+
+As of Halide 19.0.0, we provide binary wheels on PyPI which include the Python
+bindings and the C++/CMake package for native development. Full releases may be
+installed with `pip` like so:
+
+```shell
+$ pip install halide
+```
+
+Every commit to `main` is published to Test PyPI as a development version and
+these may be installed with a few extra flags:
+
+```shell
+$ pip install halide --pre --extra-index-url https://test.pypi.org/simple
+```
+
+Currently, we provide wheels for: Windows x86-64, macOS x86-64, macOS arm64, and
+Linux x86-64. The Linux wheels are built for manylinux_2_28, which makes them
+broadly compatible (Debian 10, Ubuntu 18.10, Fedora 29).
+
+## Homebrew
+
+Alternatively, if you use macOS, you can install Halide via
+[Homebrew](https://brew.sh/) like so:
+
+```
+$ brew install halide
+```
+
 ## Binary tarballs
 
 The latest version of Halide can always be found on GitHub
 at https://github.com/halide/Halide/releases
 
-We provide binary releases for many popular platforms and architectures, 
-including 32/64-bit x86 Windows, 64-bit macOS, and 32/64-bit x86/ARM
+We provide binary releases for many popular platforms and architectures,
+including 32/64-bit x86 Windows, 64-bit x86/ARM macOS, and 32/64-bit x86/ARM
 Ubuntu Linux.
 
+The macOS releases are built using XCode's command-line tools with Apple Clang
+500.2.76. This means that we link against libc++ instead of libstdc++. You may
+need to adjust compiler options accordingly if you're using an older XCode which
+does not default to libc++.
+
+We use a recent Ubuntu LTS to build the Linux releases; if your distribution is
+too old, it might not have the requisite glibc.
+
```

Comment:
Add "nightly builds of Halide and the LLVM versions we use in CI are also available at https://buildbot.halide-lang.org/"

Reply:
Done!